components/PageLink.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { Button } from '@dts-stn/service-canada-design-system'
+
+export default function PageLink(props) {
+  const router = useRouter()
+  return (
+    <>
+      <div className="pt-2 md:pt-4 my-8 border-t border-gray-light ">
+        <h2 className="font-display font-bold text-[32px] md:text-[36px]">
+          {props.lookingForText}
+        </h2>
+        <div className="font-body mb-10  text-xl">
+          <span className=" text-gray-darker" id={`link-for-${props.linkText}`}>
+            {props.accessText}{' '}
+          </span>
+          <Link href={props.href}>
+            <a className="text-blue-default hover:text-blue-hover visited:text-purple-medium underline">
+              {props.linkText}
+            </a>
+          </Link>
+        </div>
+
+        <Button
+          id={props.buttonId}
+          onClick={() => router.push('/home')}
+          styling={props.styling}
+          text={props.buttonText}
+        />
+      </div>
+    </>
+  )
+}
+
+PageLink.propTypes = {
+  // Props for the text and link
+  lookingForText: PropTypes.string,
+  linkText: PropTypes.string,
+  accessText: PropTypes.string,
+  href: PropTypes.string,
+  // Props for the Button
+  id: PropTypes.string,
+  buttonId: PropTypes.string,
+  styling: PropTypes.string,
```

Comment:
Maybe want to give this prop a more specific name at the component level. Styling for what? Is it tailwind CSS or style type?

Reply:
The styling is coming from the DS. that is what they called if for the style of the button.